src/action/bandaidset.rs

```diff
@@ -65,7 +64,7 @@ impl FirstAidKit {
             span.end.column
         );
 
-        if span.is_multiline() {
+        if span.is_multiline() || replacement.lines().count() > 1 {
```

Comment:
So you are checking if either is multiline, but what's the purpose? I would expect that we do not need this check, but just pick one of the iterators and consume it entirely, and drive the second iterator manually, any excess elements of the second are handled as `Insertion` / `Deletion` (depending on the definition of your first and second iterator).

Does that make sense?

Reply:
I think you already impl'd it like that, but the `if` clause around the first part is kind of pointless.

Reply:
Yes should not matter

Reply:
Then lets put it to rest :wastebasket: :smiley: 

less code, less :bug:s :)

Reply:
Actually, that may avoid some performance overhead since most replacements and spans will only be single-lined. That means we don't need to compute all that stuff inside this if clause.